### ISSUE TITLE
Add option to hide lemma when showing strongs number

### ISSUE THREAD

**Issue by timbze** [AndBible/and-bible#1284]
In KJV Crosswire module, when enabling strongs numbers, it will show the lemma, strongs number, and ~~part of speech~~ morphology.

~~It would be nice to have an option dialog to show/hide these 3 parts separately.~~

There is a separate option for hiding morphology. 

It would be nice to be able to hide the lemma as well. EBible's KJV module did not have lemma so I never saw it, but it always shows now cause Crosswire's version has it.

Not important for 4.0 release though.

**Current AB version:** 4.0.531-beta

**Screenshot**
![image](https://user-images.githubusercontent.com/35117769/125115290-5deef100-e0a8-11eb-9886-73c1e7064183.png)

**Comment by bvahedy** [comment]
Timmy, by speech do you mean morphology?
If so, in the windows settings as well as the workspace settings you can
turn those off.

**Comment by timbze** [comment]
Ben, ok yes there's that setting but it's always disabled for me. I wonder why?

![Screenshot_20210709-125102.png](https://user-images.githubusercontent.com/35117769/125123850-7402ae80-e0b4-11eb-9bfa-f8fd91491e9b.png)

**Comment by tuomas2** [comment]
bug maybe

**Comment by bvahedy** [comment]
That's a bug, it didn't use to do that.

**Comment by tuomas2** [comment]
I believe I might have caused that bug even today when simplifying some code...

**Comment by tuomas2** [comment]
These days morph option should not need be dependent on strongs option -- it's possible to show only morph too. Although I'm thinking it might be better to have just one "Strong & Morph" setting, which shows everything that is available.

**Comment by bvahedy** [comment]
I agree.

**Comment by timbze** [comment]
Updated OP and title.

Maybe it would be better to simply have a toggle for on/off, and then have a separate window/workspace setting for the details
* Show lemma
* Show strongs number
* Show morphology
* Show underline

**Comment by agrogers** [comment]
It's also almost unreadable with justify-text on.

How small can we make the numbers? Can we go smaller? Although it's
probably not important because we can use the new and great underline
Strongs option.

**Comment by timbze** [comment]
> Can we go smaller?

I don't think we should go smaller. I think whoever wants to use this option will be good with how it looks, cause this way it's more about the individual words anyway, IMO.

**Comment by tuomas2** [comment]
Should we hide them by default (leaving no option to show them) in 4.0?

**Comment by timbze** [comment]
> Should we hide them by default (leaving no option to show them) in 4.0?

I think that would be good and acceptable.

**Comment by agrogers** [comment]
If the original word can be turned off with a setting then that is great. I would default that setting to off - so few people would be able to use it that it wouldn't be needed.

**Comment by timbze** [comment]
> If the original word can be turned off with a setting then that is great. I would default that setting to off - so few people would be able to use it that it wouldn't be needed.

Yes, Tuomas' suggestion was to just turn it off for 4.0 and then have a setting for 4.1+. I think that's acceptable because the way 3.3 was, only the eBible module showed up so they didn't have the lemma (original word) available at all.

**Comment by tuomas2** [comment]
Now it's hidden. I'll move this ticket back to 4.1
